### PR TITLE
fix(config): add valve process before RFI flagging in GPU configs

### LINKED ISSUE #666: Add valve after rfi_vis_buffer_* before RfiDropFrame
Currently `gpu_output_n2_buffer_*` has a `valve` stage before being read by `RfiDropFrame`, but the buffers ` rfi_vis_buffer_*` don't have a `valve` stage, so it's possible that back pressure in the N2 post process pipeline can cause a backup in the GPU stage resulting in a system crash.

This addresses an issue were pulsar gaiting updates causes some nodes to crash.

### PR DESCRIPTION
**Pull request by jrs65**
Fixes #666

### CODE REVIEW

File: config/chime_science_run_gpu.yaml [hunk 1]
```diff
@@ -410,6 +410,14 @@ gpu_rfi_output_buffers:
...
   gpu_rfi_output_buffer_3:
     kotekan_buffer: standard
+  rfi_value_buffer_0:
+    kotekan_buffer: standard
+  rfi_value_buffer_1:
+    kotekan_buffer: standard
+  rfi_value_buffer_2:
+    kotekan_buffer: standard
+  rfi_value_buffer_3:
+    kotekan_buffer: standard
```
Comment (andrerenard) [inline config/chime_science_run_gpu.yaml]: I would set the `num_frames` value for these to be something much smaller.  Maybe just: `num_frames: buffer_depth`

File: config/chime_science_run_gpu.yaml [hunk 2]
```diff
@@ -458,6 +456,17 @@ gpu_rfi_output_buffers:
   gpu_rfi_output_buffer_3:
     kotekan_buffer: standard
 
+gpu_rfi_valve_buffers:
+  num_frames: buffer_depth
```
Comment (andrerenard) [inline config/chime_science_run_gpu.yaml:460]: Frame size is missing here.

File: config/chime_science_run_gpu_no_output.yaml [hunk 3]
```diff
@@ -459,6 +459,14 @@ gpu_rfi_output_buffers:
...
   gpu_rfi_output_buffer_3:
     kotekan_buffer: standard
+  rfi_value_buffer_0:
+    kotekan_buffer: standard
+  rfi_value_buffer_1:
+    kotekan_buffer: standard
+  rfi_value_buffer_2:
+    kotekan_buffer: standard
+  rfi_value_buffer_3:
+    kotekan_buffer: standard
```
Comment (andrerenard) [inline config/chime_science_run_gpu_no_output.yaml:469]: This needs to be updated to match the production config.

File: config/chime_science_run_gpu.yaml (hunk 2 above)
Comment (jrs65) [reply config/chime_science_run_gpu.yaml:460]: Yeah. I found this one out pretty rudely last night.

File: config/chime_science_run_gpu_no_output.yaml (hunk 3 above)
Comment (jrs65) [reply config/chime_science_run_gpu_no_output.yaml:469]: Ah good point.

**Pull request merged**